.github/ISSUE_TEMPLATE/2-1.0-issue.yaml

```diff
@@ -0,0 +1,95 @@
+name: "\U0001F41B 1.0 Issue"
+description: Create an issue that is required for Matter 1.0 release
+title: "[1.0] "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for reporting an issue against the Matter SDK! We need information about the bug report to follow up, so please help us out by filling out this information.
+  - type: textarea
+    id: repro
+    attributes:
+      label: Reproduction steps
+      description: "How do you trigger this bug? Please walk us through it step by step."
+      value: |
+        1.
+        2.
+        3.
+        ...
+      render: bash
+    validations:
+      required: true
+  - type: input
+    id: prevalence
+    attributes:
+      label: Bug prevalence
+      description: "How often do you or others encounter this bug?"
+      placeholder: "Example: Whenever I visit the personal account page (1-2 times a week, day, hour)"
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: GitHub hash of the SDK that was being used
+      description: Hash of the GitHub SDK used
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Platform
+      description: What platforms are affected?
+      multiple: true
+      options:
+        - ameba
+        - android
+        - cc13x2_cc26x2
+        - darwin
+        - efr32
+        - esp32
+        - freeRTOS
+        - IMX8
+        - k32w
+        - nrf connect
+        - nrf
+        - python
+        - raspi
+        - vscode
+        - windows
+        - other
+        - core
+    validations:
+      required: true
+  - type: input
+    id: platform-versions
+    attributes:
+      label: Platform Version(s)
+      description: "What platform version(s) are affected [optional]"
+      placeholder: "1.0.1"
+  - type: dropdown
+    attributes:
+      label: Type
+      description: What type of issue is this?
+      multiple: true
+      options:
+        - Test Improvement
+        - Common Cluster Logic
+        - Spec Compliance Issue
+        - Security Issue
+        - Platform Issue
+        - Core SDK Memory Issue
+        - Core SDK Crash
+        - Core SDK Performance Improvement
+        - Core SDK Interopability Issue
```

Comment:
Always add `Other` as an option?

Reply:
What is "Other" ? If it doesn't fit here, then... we shouldn't accept it?

Reply:
Happy to add more if there are more categories, but I think the point here is to only specific certain changes in.

Reply:
I think you can never predict all classes. Or would checking no checkbox be acceptable?